app/test_main.py

```diff
@@ -1,3 +1,50 @@
+import pytest
+
 from app.main import get_human_age
 
-# write your code here
+
+test_cases = [
+    (-1, -2, [0, 0]),
+    (0, 0, [0, 0]),
+    (14, 14, [0, 0]),
+    (15, 15, [1, 1]),
+    (23, 23, [1, 1]),
+    (24, 24, [2, 2]),
+    (27, 27, [2, 2]),
+    (28, 28, [3, 2]),
+    (100, 100, [21, 17])
+]
+
+
+@pytest.mark.parametrize("cat_age, dog_age, expected_result", test_cases)
+def test_get_human_age(cat_age: int,
```

Comment:
After opening the parentheses, write each the arguments on a new line, including the first one, the closing bracket is also written on a new line